providers/curation/github.js

```diff
@@ -193,7 +193,13 @@ class GitHubCurationService {
     const target_url = `https://dev.clearlydefined.io/curation/${pr.title}/pr/${pr.number}`;
     try {
       return github.repos.createStatus({
-        owner, repo, sha, state, description, target_url, context: 'ClearlyDefined'
+        owner,
```

Comment:
this is a good example of the line length topic.  this expression is about as simple as they come yet it is formatted over 9 lines rather than 1 (or 2-3). 

Not that in this case prettier split the lines because the original had the object split over two lines. If the original is all on one line, prettier leaves it.  But, if one of the property names was even one char longer, we get the same many line format out.

In this case we can get it to 103 chars by declaring `const context = 'ClearlyDefined'` and then using that in the last element of the literal.

Reply:
Yes, it's not perfect and I agree with you that this should be on one line. Perhaps this will be improved in later versions of prettier. There appears to be some discussion on the issue here https://github.com/prettier/prettier/issues/2716

Might be good to weigh in there.

Reply:
> Not that in this case prettier split the lines because the original had the object split over two lines. If the original is all on one line, prettier leaves it.

Yeah I've noticed at times it'll leave it in it's original form.